### ISSUE TITLE
Add a way to set categories on workflow

### ISSUE THREAD

**Issue by jdotcms** [dotCMS/core#23009]
Currently you can recovery categories from the api, but you can not set the categories to the contentlet in order to be saved.

Acceptance criteria:

After the change a documented approach should exists to set the categories into the contentlet on workflow as a velocity script

Customer ticket: https://dotcms.zendesk.com/agent/tickets/108490

**Comment by jdotcms** [comment]
[PR](https://github.com/dotCMS/core/pull/23010/files)

**Comment by jdotcms** [comment]
In order to make it works you have to:
1) create a content type with categories 
<img width="1036" alt="Screen Shot 2022-09-16 at 1 35 49 PM" src="https://user-images.githubusercontent.com/19578350/190708417-2012066a-4818-4580-b26e-7a4c727bf6e2.png">

2) create a workflow action (velocity script actionlet)
<img width="826" alt="Screen Shot 2022-09-16 at 1 32 40 PM" src="https://user-images.githubusercontent.com/19578350/190709283-e000365f-3aaa-4969-a51f-36d16b8fd59a.png">

```

#set($parentCategory = $categories.getCategoryByKey('ageOrGender'))
#set($cats= $categories.getChildrenCategories($categories.getCategoryByKey('ageOrGender')))
#set($contentDeps = $workflow.copyContentletDependenciesBuilder().categories($cats).build())
$workflow.setContentletDependencies($contentDeps))
```

Create a new type and fire the new action, the categories of age or gender will be assigned to the contentlet

**Comment by erickgonzalez** [comment]
This seems that is already merged in 22.12 but it wasn't QA, so marking as a Priority just to do QA
